scripts/ReserveMargin.py

```diff
@@ -34,22 +35,12 @@ def main():
         'NS':1.15,
         'PEI':1.15}
 
-    #Years to Print over
-    years = range(2019,2051,1)
-
     # List of fuels to tag
-    fuelTag = ['ELC']
+    # fuelTag = ['ELC']
 
     # List of technologies to tag
-    #techTags = ['HYD','BIO','CCG','CTG','URN','COA','COC','WND', 'SPV']
-    techTags = ['HYD','BIO','CCG','CTG','URN','COA','COC']
-
-    #Dictionary holds month to season Mapping 
-    seasons = {
-        'W':[1, 2, 3],
-        'SP':[4, 5, 6],
-        'S':[7, 8, 9],
-        'F':[10, 11, 12]}
+    # TECH_TAGS = ['HYD','BIO','CCG','CTG','URN','COA','COC','WND', 'SPV']
+    TECH_TAGS = ['HYD','BIO','CCG','CTG','URN','COA','COC']

```

Comment:
Do you think these would be a good to store in the config file?

Reply:
I'll add a non-dispatchable techs list to the file, and we can use that